app/src/main/res/values/map_api.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="maps_api_key" translatable="false">AIzaSyD5cM2d_MF2eRdZbQ1UENjAZnaaQDXBTt0</string>
```

Comment:
Uploading API keys to github is a bad practice....you should probably remove this, revoke the key, and add this file to the gitignore. Then you can distribute the api key to us via some private channel.

Reply:
yea there is a way to hide it from github